packages/cozy-flags/package.json

```diff
@@ -31,6 +31,9 @@
     "react": "16.12.0"
   },
   "jest": {
+    "globals": {
+      "window": {}
+    },
```

Comment:
We don't want to have this `window` stuff. cozy-flags is also used in a node env. It should work without the need to rely on window.

Reply:
En effet, la version de `cozy-client` étant assez en retard sur `cozy-flags`, ce derniers ne bénéficiait pas des fix autour de ce sujet.
Une simple maj de `cozy-client` suffit à corriger le problème, et donc à ne plus avoir besoin de ce changement!